generators/server/templates/src/main/java/package/config/_CacheConfiguration.java

```diff
@@ -257,6 +257,9 @@ public HazelcastInstance hazelcastInstance(JHipsterProperties jHipsterProperties
         }
         <%_ } _%>
         config.getMapConfigs().put("default", initializeDefaultMapConfig());
+        
+        // Full reference is available at: http://docs.hazelcast.org/docs/management-center/3.9/manual/html/Deploying_and_Starting.html
+        config.setManagementCenterConfig(initializeDefaultManagementCenterConfig(jHipsterProperties))
```

Comment:
Missing a `;` here 😉

Reply:
sorry,it was careless, i edit it by notepad++, can you tell me what ide do you use for `generator-jhipster` project?